sentences/en/_common.yaml

```diff
@@ -376,15 +376,15 @@ expansion_rules:
   are_any: "[<are> ]<any>"
   how_many: "how many[ of the]"
   brightness: "{brightness}[[ ]%| percent]"
-  light: "[the ](light|lights|lighting|lamp|lamps)"
-  turn: "(turn|switch|change)"
+  light: "[the ](light|lights|lighting|lamp|lamps|bulb|bulbs|light fixture|light fixtures|light switch|light switches)"
+  turn: "(turn|switch|change|bring)"
   temp: "(temp|temperature)"
   temperature: "{temperature}[([ ]°[ ][{temperature_unit}])|( degrees[ {temperature_unit}])]"
   open: "(open|raise|lift)[ up]"
   close: "(close|shut|lower)[ (up|down)]"
   set: "(set|make|change|turn)"
   numeric_value_set: "(set|change|turn[ (up|down)]|increase|decrease|make)"
-  in: "(in|on|at|of)"
+  in: "(in|on|at|of|across|around|throughout)"
```

Comment:
Is there any way "around" could mean "the areas around the one i'm mentioning"? If so, let's drop it.

Reply:
Yeah, I would imagine that if I said "around the living room", it would affect rooms next to the living room, but not the living room itself.

Reply:
That interpretation doesn't really resonate with me. Like, If I say "turn on the lights all around my house", I'm generally referring to inside, not perimeter. More like, within the perimeter.

Similar to how someone saying "Let's go around the room" typically refers to within the room, but inclusive of everyone, rather than outside the perimeter.

Reply:
I get your point. The questions is: is "turn on the lights around the bedroom" (1) that common in terms of usage of "around" with any meaning and (2) widespread enough in terms of "around" meaning "inside the room" to mandate the change?

Reply:
Well, I think so. :) For what it's worth, if I ask Siri to "Turn the lights on around the kitchen", all lights in my kitchen go on. Not that it means we _must always_ do what Siri or others do, but it does at least indicate I'm not just an outlier.

Reply:
Alrighty, then. Rebase, fix the tests and let's get it merged